src/pages/playlist/playlist.js

```diff
@@ -0,0 +1,66 @@
+import axios from "axios"
+import { useEffect } from "react"
+import './playlist.css'
+import { Navbar } from "../../components/navbar/navbar"
+import { Sidebar } from "../../components/sidebar/sidebar"
+import { usePlaylist } from "../../context/playlist-context"
+import { Link,useParams } from "react-router-dom"
+
+export const PlayList=()=>{
+    const {playlist,setPlaylist}=usePlaylist()
+    
+
+
+    useEffect(()=>{
+        getPlayList()
+    },[])
+
+    const getPlayList= async()=>{
+        const token=localStorage.getItem("user")
+        try {
+            const response=await axios.get(`/api/user/playlists`,{
+                headers: {
+                  authorization: token, // passing token as an authorization header
+                },
+              })
+              setPlaylist(response.data.playlists);
+        } catch (error) {
+            console.log(error);
+        }
+    }
+
+
+    const deletePlaylistHandler= async (id)=>{
```

Comment:
put it into separate file so that it looks clean

Reply:
ok